pkg/plugins/platform/local_test.go

```diff
@@ -131,6 +131,24 @@ func TestLocalStart(t *testing.T) {
 
 		cleanup()
 	}
+
+	// try to start a site that is already running
+	err = app.Init(TestSites[0].Dir)
+	assert.NoError(err)
+
+	err = app.Start()
+	assert.EqualError(err, fmt.Sprintf("a site with the name %s is already running", TestSites[0].Name))
```

Comment:
IMHO, this tells the user that there is a problem without necessarily pointing them to a resolution. For example, I would have to remember to run `ddev config` to change the site name.